src/Core/src/AbstractApi.php

```diff
@@ -99,6 +100,17 @@ public function __construct($configuration = [], ?CredentialProvider $credential
     abstract protected function getSignatureScopeName(): string;
 
     final protected function getResponse(Request $request, ?RequestContext $context = null): Response
+    {
+        return new Response(
+            $this->createHttpResponse($request, $context),
+            $this->httpClient
+        );
+    }
+
+    /**
+     * @throws \Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface
+     */
+    final protected function createHttpResponse(Request $request, ?RequestContext $context = null): ResponseInterface
```

Comment:
New method to get the HttpResponse without wrapper
I was stuck because of https://github.com/async-aws/aws/blob/master/src/Core/src/Response.php#L46-L51